charts/kaito/workspace/crds/kaito.sh_workspaces.yaml

```diff
@@ -29,8 +29,11 @@ spec:
     - jsonPath: .status.conditions[?(@.type=="InferenceReady")].status
       name: InferenceReady
       type: string
-    - jsonPath: .status.conditions[?(@.type=="WorkspaceReady")].status
-      name: WorkspaceReady
+    - jsonPath: .status.conditions[?(@.type=="JobStarted")].status
+      name: JobStarted
```

Comment:
So this job started column will be present even for inference workloads?

Reply:
yes